packages/@aws-cdk/aws-glue/test/test.connection.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from '@aws-cdk/assert';
+import { Stack } from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import * as glue from '../lib';
+import { Vpc } from '@aws-cdk/aws-ec2';
+
+export = {
+  'creates a connection'(test: Test) {
+    const stack = new Stack();
+
+    new glue.Connection(stack, 'Connection', {
+        connectionProperties: {},
+    });
+
+    expect(stack).toMatch({
```

Comment:
I think you want `expect(stack).toHaveResourceLike(...)`.... right now, these tests don't actually verify anything.

Reply:
Yea im still working on tests.